spec/support/acceptance_spec_helpers.rb

```diff
@@ -45,13 +45,13 @@ def migrate_down(expected_value)
 
   class MigrationUpEquals < RSpec::Matchers::BuiltIn::Eq
     def matches?(subject)
-      super(subject[0].gsub(/, +([a-z_]+:)/i, ', \1')) # normalize multiple spaces to one
+      super(subject[0].strip.gsub(/, +([a-z_]+:)/i, ', \1').gsub(/\n+/, "\n")) # normalize multiple spaces and newlines to one
     end
   end
 
   class MigrationDownEquals < RSpec::Matchers::BuiltIn::Eq
     def matches?(subject)
-      super(subject[1].gsub(/, +([a-z_]+:)/i, ', \1')) # normalize multiple spaces to one
+      super(subject[1].strip.gsub(/, +([a-z_]+:)/i, ', \1').gsub(/\n+/, "\n")) # normalize multiple spaces and newlines to one
```

Comment:
I made these changes so tests wouldn't fail based on whitespace. I didn't want to waste time on that. We can always tune whitespace some other time.